okhttp/src/test/java/okhttp3/internal/concurrent/TaskFaker.kt

```diff
@@ -28,6 +25,26 @@ import org.assertj.core.api.Assertions.assertThat
  * [advanceUntil].
  */
 class TaskFaker {
+  @Suppress("NOTHING_TO_INLINE")
+  internal inline fun Any.assertThreadHoldsLock() {
```

Comment:
I missed this previously, because we include these files via gradle source sets, so they don't get access to internal.